src/zc/buildout/download.py

```diff
@@ -31,17 +31,28 @@ class PatchedURLopener(FancyURLopener):
 
 except ImportError:
     # Python 2
+    import base64
     from urlparse import urlparse
-
+    from urlparse import urlunparse
     import urllib2
 
-    def urlretrieve(url, path):
-        """Work around Python issue 24599
+    def urlretrieve(url, tmp):
```

Comment:
Why have you renamed "path" to "tmp"? It might be the right decision, but I'd like to know the reason :-)